x-pack/plugins/security_solution/public/management/components/console/types.ts

```diff
@@ -46,8 +46,10 @@ export interface CommandArgDefinition {
    *   the value entered will first be `trim()`'d.
    * - `number`: user's value will be converted to a Number and ensured to be a `safe integer`
    * - `number-greater-than-zero`: user's value must be a number greater than zero
+   * - `truthy`: The argument must have a value and the values must be "truthy"
+   *   (no `null` or `undefined`). Note that `0` (zero) will be considered `truthy`
```

Comment:
What I understood from the comment is `false`/`''` are also `truthy` same as `0`.

Reply:
hmmm. no, the implemenation looks like this:

```
if (typeof argValue !== 'number' && !argValue) {
```

So `number` of `0` is truthy, but everything else is not. so `false`, `''` would fail validation.

I might have perhaps been too explicit on this `thruthy` validator. Maybe I should just remove the `typeof argValue !== 'number` and just test that value evaluates to a boolean true?

if someone wants to ensure its not an empty string, then they could use `mustHaveValue: 'non-empty-string'`. And if they want to ensure the number zero can be used, then they would set the `mustHaveValue: 'number'`. 

What do you think?